vtr/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
 
 source:
   git_url: https://github.com/SymbiFlow/vtr-verilog-to-routing.git
-  git_rev: master+wip
+  git_rev: 253f75b6d
```

Comment:
Both `vtr` and `vtr-gui` packages should be using the same `git_rev` value.

Otherwise LGTM, feel free to merge.

Reply:
I set this value in both `vtr` and `vtr-gui` packages, as this version is currently used in symbiflow-examples.